packages/react-components/react-datepicker-compat/src/components/DatePicker/DatePicker.test.tsx

```diff
@@ -72,7 +69,7 @@ describe('DatePicker', () => {
 
   it('should call onSelectDate even when required input is empty when allowTextInput is true', () => {
     const onSelectDate = jest.fn();
-    const result = render(<DatePicker isRequired allowTextInput onSelectDate={onSelectDate} />);
+    const result = render(<DatePicker allowTextInput onSelectDate={onSelectDate} />);
```

Comment:
Don't you need to pass required here? Alternatively, you could change the test description.